jax/scipy/linalg.py

```diff
@@ -41,3 +41,5 @@
 from jax._src.lax.polar import (
   polar_unitary as polar_unitary,
 )
+
+from jax._src.third_party.scipy.linalg import funm
```

Comment:
We need to `import funm as funm` otherwise it will be treated as a private method by type checkers.